### ISSUE TITLE
[electrophysiology_browser] Can not load session page (test VM)

### ISSUE THREAD

**Issue by driusan** [aces/Loris#6226]
**Describe the bug**
When accessing the EEG browser on the test VM, the page doesn't load after clicking on "all types" or "raw" and gets stuck on a "Loading" spinner.

The javascript console says:

```
Error: "Internal Server Error"
    fetchData https://test-loris-dev.loris.ca/electrophysiology_browser/js/electrophysiologySessionView.js:2234
```

I don't have any access to the apache logs to see what the cause of the internal server error is.

**To Reproduce**
Log in and visit a URL such as: https://test-loris-dev.loris.ca/electrophysiology_browser/sessions/166?outputType=raw

**What did you expect to happen?**
Data would load in some way.

**Comment by kongtiaowang** [comment]
I think this is a bug.
[client 192.168.122.1:46552] PHP Fatal error:  Uncaught NotFound: Could not load module `electrophysiology_browser`: file `/var/www/Loris/modules/electrophysiology_browser/php/models/electrophysiofile.class.inc` does not exist in /var/www/Loris/php/libraries/Module.class.inc:141\nStack trace:\n#0 [internal function]: Module::{closure}('LORIS\\\\electroph...')\n#1 /var/www/Loris/modules/electrophysiology_browser/php/sessions.class.inc(242): spl_autoload_call('LORIS\\\\electroph...')\n#2 /var/www/Loris/modules/electrophysiology_browser/php/sessions.class.inc(164): LORIS\\electrophysiology_browser\\Sessions->getFilesData(174, 'all_types')\n#3 /var/www/Loris/modules/electrophysiology_browser/php/sessions.class.inc(129): LORIS\\electrophysiology_browser\\Sessions->getSessionData(174, 'all_types')\n#4 /var/www/Loris/src/Middleware/UserPageDecorationMiddleware.php(234): LORIS\\electrophysiology_browser\\Sessions->handle(Object(Laminas\\Diactoros\\ServerRequest))\n#5 /var/www/Loris/src/Middleware/PageDecorationMiddleware.php(56): LORIS\\Middleware\\UserPageDecorationMiddleware->proce in /var/www/Loris/php/libraries/Module.class.inc on line 141, referer: https://test-loris-dev.loris.ca/electrophysiology_browser/sessions/174

**Comment by cmadjar** [comment]
@kongtiaowang could you bring that up to testing, retest and close the issue if resolved.

**Comment by christinerogers** [comment]
@cmadjar this issue resolved  by #6238 (they should have been linked before) 

I asked @kongtiaowang this morning to pull this PR/branch onto the test-dev VM (it's there now) -- I tested it and it's resolved.  
Sorry for not keeping you in the loop.

**Comment by christinerogers** [comment]
Resolved by #6238 - closing this issue.
